ansible/templates/slurm.conf.j2

```diff
@@ -48,10 +48,6 @@ RebootProgram="{{ slurm_reboot_program }}"
 ResumeTimeout={{ slurm_resume_timeout }}
 JobFileAppend=1
 
-# Fallback memory allocation values
```

Comment:
Mon ikke det er sikkert at sætte mem defaults på igen?

Reply:
Det tænker jeg ikke - det ødelægger staging og fungere i praksis som en prop for udviling

Reply:
Hvordan bør vi så gøre på prod.? Har vi ikke andre valgmuligheder end at instruere brugere i altid at angive "--mem"?

Reply:
Jeg synes vi skal ta det med her over i https://github.com/aau-claaudia/aicloud/issues/22 - det er forholdsvis urelateret til gpu labels

Reply:
Men fair nok @ThomasA vi kan egentlig godt sætte det ind og så alligevel gå igang med #22 umiddelbart bagefter - det kan være det samme - og giver forhåbentligt mindre kunde kontakt :)

Reply:
således..